### DISCUSSION TITLE
Semester 4 Week 2

### DISCUSSION THREAD

**Discussion by BenjaminB109** [SE-TINF22B6/Underwatch#155] [Announcements]
# Design
This week we continued with the design of the maps, among other things. We have decided on another graphic style. This one represents the outside world of the dungeon. There are orcs running around as enemies and they all have movement animations for 4 directions. On  map 1, you are spawned at the top left and have to go to the bottom left to get to the entrance of the dungeon. 
### Map 1
![grafik](https://github.com/SE-TINF22B6/Underwatch/assets/122724021/db333c96-b3f0-4dc9-8e3d-78d881085018)

Map 2 is the first dungeon map. It still uses the familiar style. The opponents in the dungeon are fearsome skeletons. The most powerful of them have a red hat on. It has not yet been decided whether Map 2 will retain its current dimensions.

### Map 2
![grafik](https://github.com/SE-TINF22B6/Underwatch/assets/122724021/1f530b20-e02c-4403-8dba-0e947c108be4)

# Website
We also maade some achievments with the website. The first thing is that the scoreboard shows all the data from the database now. The other achievment is a working Champions-page, that shows the best three players ever. In the future this should be extended so that there is a possibility to also have a look at the best players in the current month.

![Screenshot 2024-04-24 143016](https://github.com/SE-TINF22B6/Underwatch/assets/123720714/0110e6d7-ad83-43ed-bc6b-78b937ea7afc)
![Screenshot 2024-04-24 143242](https://github.com/SE-TINF22B6/Underwatch/assets/123720714/a7d52ee7-aff6-498e-a631-384ca3b201b8)

**Comment by giomskii** [comment]
Good Day Team Underwatch,
both maps look really impressive with unique structures. Are they interconnected during the actual gameplay or seperate?
Your website progress also looks quite good! Will you also add some additional details to the champions page like the overall score for each top player?
Nonethless, keep up the good work! - Team DHBWHub 👍🏼

**Comment by NicolasVien** [comment]
Your maps design looks amazing! Great progress on the website, I like the idea of a global scoreboard ! Im looking forward to tthe implementation. Keep up the great work !

**Comment by Shiny1708** [comment]
The maps look very promising, especially the first one. The second map looks a bit empty from the screenshot so you could try to include more decorations on there. The Achievements are a great idea and the design looks comfy, I personally would like to see the usernames take up more space tho since they feel a bit small compared to the pillars they are on top of.

**Comment by LeonmAndreesen** [comment]
We really like your game. You put some efforts into the opponments. It is cool to see that you make a diiference between the strength of the single enemies. Furthermore it is cool to see that you do good work on the maps.

**Comment by Challenger3010** [comment]
It seems like you put a lot of effort into your game. We look forward to playing it. Will there be more than 2 maps and maybe more playable characters?

Keep the work up - Team CookHub
